README.md

```diff
@@ -185,11 +185,97 @@ python -m CHAID tests/data/titanic.csv fare sex embarked --max-depth 4 --min-par
 {'node': 7, 'rules': [{'variable': 'sex', 'data': ['male']}, {'variable': 'embarked', 'data': ['S']}]}
 ```
 
-
 Parameters
-----------
+-------
 Run `python -m CHAID -h` to see description of command line arguments
 
+How to Read the Tree
+-------
+
+We'll start with a real world example using the titanic dataset.
+
+Run `python -m CHAID tests/data/titanic.csv survived sex embarked --max-depth 4 --min-parent-node-size 2 --alpha-merge 0.05` after placing an ipdb statement on like 55 on __main__.py as in the example below. The parameters mean max depth two 4 levels, a minimum parent node size threshold to 2 and merge the groups if the p-value is greater than 0.05 when comparing the groups.
+
+```
+82        tree = Tree.from_pandas_df(data, independent_variables,
+83                                   nspace.dependent_variable[0],
+84                                   variable_types=types, **config)
+---> 85   import ipdb; ipdb.set_trace()
+86    
+87        if nspace.classify:
+88            predictions = pd.Series(tree.node_predictions())
+89            predictions.name = 'node_id'
+90            data = pd.concat([data, predictions], axis=1)
+91            print(data.to_csv())
+92        elif nspace.predict:
+```
+
+Running `tree.print_tree()` gives:
+
+``` python
+([], {0: 809, 1: 500}, (sex, p=1.47145310169e-81, score=365.886947811, groups=[['female'], ['male']]), dof=1))
+├── (['female'], {0: 127, 1: 339}, (embarked, p=9.17624191599e-07, score=24.0936494474, groups=[['C', '<missing>'], ['Q', 'S']]), dof=1))
+│   ├── (['C', '<missing>'], {0: 11, 1: 104}, <Invalid Chaid Split>)
+│   └── (['Q', 'S'], {0: 116, 1: 235}, <Invalid Chaid Split>)
+└── (['male'], {0: 682, 1: 161}, (embarked, p=5.017855245e-05, score=16.4413525404, groups=[['C'], ['Q', 'S']]), dof=1))
+    ├── (['C'], {0: 109, 1: 48}, <Invalid Chaid Split>)
+    └── (['Q', 'S'], {0: 573, 1: 113}, <Invalid Chaid Split>)
+```
+
+as show above. The first line is the root node. This means that all the data is present in this node. And the vertical bars originating from a node represents paths to that node's children.
```

Comment:
It's absolutely fine to start a sentence with 'And'. But will change to appease.